charts/newrelic-infrastructure/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart to deploy the New Relic Infrastructure Agent as a DaemonSet
 name: newrelic-infrastructure
-version: 1.3.2
-appVersion: 1.26.4
+version: 1.3.3
+appVersion: 1.26.1
```

Comment:
Why are you changing the appVersion?

Reply:
ups that is a mistake from the merge from master commit i think